src/lmic/lmic.c

```diff
@@ -1551,6 +1552,12 @@ static bit_t processJoinAccept (void) {
 
     if (seErr != LMIC_SecureElement_Error_OK)
         return processJoinAccept_badframe();
+    
+    LMIC_SecureElement_getAppSKey(&AppSKey, 0);
+    LMIC_SecureElement_getNwkSKey(&NwkSKey, 0);
+
+    memcpy(LMIC.artKey, AppSKey.bytes, sizeof(LMIC.artKey));
```

Comment:
No. LMIC.artKey and so forth are going to be removed. They're not needed.

Reply:
Removed LMIC.artKey and LMIC.nwkKey.